snuba/datasets/metrics_messages.py

```diff
@@ -61,3 +66,38 @@ def value_for_counter_message(message: Mapping[str, Any]) -> Mapping[str, Any]:
     ), f"{ILLEGAL_VALUE_IN_COUNTER} {INT_FLOAT_EXPECTED}: {value}"
 
     return {"metric_type": OutputType.COUNTER.value, "count_value": value}
+
+
+def aggregation_options_for_set_message(_: Mapping[str, Any]) -> Mapping[str, Any]:
+    return {
+        "materialization_version": 1,
+        "granularities": [
+            GRANULARITY_ONE_MINUTE,
+            GRANULARITY_ONE_HOUR,
+            GRANULARITY_ONE_DAY,
+        ],
```

Comment:
Does this need to be overridable for different metric types?

Reply:
Yes, I was thinking that we can parse the aggregation option enum here, and some enum might specify that we don't need GRANULARITY_ONE_MINUTE, so we can just omit it here